### PR TITLE
cloudformation for iac to S3 complete

### PR DESCRIPTION
**Pull request by eric-nord**
Updated a few file names to keep consistent and updated related script.

### CODE REVIEW

File: cloudformation/account/account-setup.sh [hunk 1]
```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+aws cloudformation create-stack \
+--stack-name account-setup \
+--template-url "https://s3-us-west-2.amazonaws.com/byu-818272543125-us-west-2-cloudformation-templates/iac/cloudformation/account/iac-sync-S3.yaml" \
+--disable-rollback \
```
Comment (pauldeden) [inline cloudformation/account/account-setup.sh]: Why disable rollback here?

File: cloudformation/account/account-setup.sh [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+--capabilities CAPABILITY_NAMED_IAM \
+--parameters \
+ParameterKey=team,ParameterValue=AD-Student-Life-Applications \
```
Comment (pauldeden) [inline cloudformation/account/account-setup.sh]: Are these Parameter tags defaults for the account that can be overridden by more specific tags on individual resources?

File: cloudformation/account/account-setup.yaml [hunk 3]
```diff
@@ -0,0 +1,107 @@
...
+    Default: dev
+  dataSensitivity:
+    Type: String
+    Description: (required) - see https://byuoit.atlassian.net/wiki/display/OAPP/Tagging
+    AllowedValues:
+      - public
+      - internal
+      - confidential
+      - highly confidential
+    Default: public
```
Comment (pauldeden) [inline cloudformation/account/account-setup.yaml]: The default should be confidential (as defined here: https://infohub-dev.byu.edu/resources/classification)

File: cloudformation/account/account-setup.sh (hunk 2 above)
Comment (pauldeden) [inline cloudformation/account/account-setup.sh]: If that is the case, then I like it.

File: cloudformation/account/account-setup.sh (hunk 1 above)
Comment (eric-nord) [inline cloudformation/account/account-setup.sh]: Allows for better diagnostics in the case of a creation failure.

Comment (pauldeden) [inline cloudformation/account/account-setup.sh]: OK

File: cloudformation/account/account-setup.sh (hunk 2 above)
Comment (eric-nord) [inline cloudformation/account/account-setup.sh]: These parameter tags are where they are overwritten.
The sh scripts are meant to customize based on utilization

**Pull request merged**